extensions/ccui/uiwidgets/UIButton.js

```diff
@@ -33,7 +33,7 @@
  * @property {String}   titleFont               - The content string font of the button title
  * @property {Number}   titleFontSize           - The content string font size of the button title
  * @property {String}   titleFontName           - The content string font name of the button title
- * @property {cc.Color} titleFontColor          - The content string font color of the button title
+ * @property {cc.Color} titleColor          - The content string font color of the button title
```

Comment:
Should probably fix spacing so description aligns with the others

Reply:
I will fix it